syne_tune/optimizer/schedulers/searchers/bayesopt/models/estimator.py

```diff
@@ -156,14 +156,14 @@ def transform_state_to_data(
     if state.pending_evaluations:
         # In this case, y becomes a matrix, where the observed values are
         # broadcast
-        cand_lst = [
-            hp_ranges.to_ndarray(config) for config in state.pending_configurations()
-        ]
+        cand_lst = []
```

Comment:
This change needs some comment, I am not sure this works out.

Reply:
This was commited by mistake, already removed in the new commit.